db/migrate/20220707204130_add_post_id_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostIdToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :post_id, :integer
+  end
+end
```

Comment:
- [ ] The same logic as in the review up :point_up: this migration is not adding any references. But this is adding though `rails g migration addPostToComments post:references ` it will generate a migration like this :point_down: 

```rb
class AddPostToComments < ActiveRecord::Migration[7.0]
  def change
    add_reference :comments, :post, null: false, foreign_key: true
  end
end
```